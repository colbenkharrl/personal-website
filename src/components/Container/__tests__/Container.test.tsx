import { render, screen } from '@testing-library/react';
import { Container } from '../';
import meta from '../__stories__/Container.stories';

it('Container matches snapshot (regression test)', () => {
  render(<Container {...meta.args} />);

  screen.getByText('This is some content.');

  expect(screen.getByTestId('Container')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
