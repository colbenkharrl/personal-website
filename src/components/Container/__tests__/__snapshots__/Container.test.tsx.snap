// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container matches snapshot (regression test): Full component snapshot. 1`] = `
<div
  class="sm:px-8"
  data-testid="Container"
>
  <div
    class="mx-auto w-full max-w-7xl lg:px-8"
  >
    <div
      class="relative px-4 sm:px-8 lg:px-12"
    >
      <div
        class="mx-auto max-w-2xl lg:max-w-5xl"
      >
        <p>
          This is some content.
        </p>
      </div>
    </div>
  </div>
</div>
`;
